blogs/AngularBootstrap.md

```diff
@@ -0,0 +1,136 @@
+This post walks through bootstrapping an angular app asynchronously. You may find yourself needing to bootstrap angular asynchronously if you need to load system wide configs for example.
+
+We'll cover the following steps to asynchronously boostrap angular:
```

Comment:
Is it worth defining what it means to "bootstrap" angular? Based on reading this, my understanding is that this gives you a snippet for everything you need to do between the initial javacsript/html load and having a fully-functional angular app on the page, presumably trying to avoid FOUC and showing a nice loading animation. Is that right?